service/controller/aws/v17/version_bundle.go

```diff
@@ -9,7 +9,7 @@ func VersionBundle() versionbundle.Bundle {
 		Changelogs: []versionbundle.Changelog{
 			{
 				Component:   "node-exporter",
-				Description: "Disabled ipvs collector.",
+				Description: "Disable ipvs collector.",
```

Comment:
AIUI this is correct and I prefer this style. But please also fix the docs.

https://github.com/giantswarm/fmt/blob/master/releases/how_write_changelog_message.md

cc @xh3b4sd